README.md

```diff
@@ -15,24 +15,16 @@
     - queries the AD server to create a new counter, increase it, and fetches the latest state from the Synchronizer
 
 
-## API endpoints
+## Usage
+### Requirements
+Required software: [curl](https://curl.se), [git](https://git-scm.com), [rust](https://rust-lang.org), [go](https://go.dev), [tmux](https://github.com/tmux/tmux), [jq](https://github.com/jqlang/jq)
 
-### AD Server
+Copy the `.env.default` file into `.env`, and set the `PRIV_KEY` and `RPC_URL` values.
```

Comment:
The `load_dotenv` function loads both `.env.default` and `.env`, so theres' no need to copy the files.  Instead I would suggest just creating a `.env` file with the `PRIV_KEY` entry (to overwrite the empty one in `.env.default`.

Perhaps the mechanism of having two files where first one is loaded and then the other one replacing variables if they collide, is not very intuitive?

Reply:
Actually, added this line because the `full-flow.sh` will update the `.env` file to put there the latest slot number at the `AD_GENESIS_SLOT` variable, so if the user is using only the `.env.default`, the genesis slot there will not be updated for running the tests.
Another option would be that the `full-flow.sh` file updates also the `.env.default` file, so that the update of the value of `AD_GENESIS_SLOT` happens both on `.env` and `.env.default`.